app/javascript/components/analytics/widgets/GeographicSummary.js

```diff
@@ -11,53 +11,54 @@ import { insularAreas } from '../mapData';
 import { stateOptions } from '../../../data/stateOptions';
 import CountyLevelMaps from './CountyLevelMaps';
 
-const MAX_DAYS_OF_HISTORY = 10; // only allow the user to scrub back N days from today
-const INITIAL_SELECTED_DATE_INDEX = 9;
 const TERRITORY_GEOJSON_FILE = 'usaTerritories.json';
-let JURISDICTIONS_NOT_IN_USE = {
-  states: [],
-  insularAreas: [],
-};
 
 class GeographicSummary extends React.Component {
   constructor(props) {
     super(props);
-    this.analyticsData = this.parseAnalyticsStatistics();
-    this.jurisdictionsPermittedToView = this.obtainJurisdictionsPermittedToView();
-    this.obtainJurisdictionsNotInUse();
-    if (this.analyticsData.exposure[Number(INITIAL_SELECTED_DATE_INDEX)]) {
-      this.state = {
-        selectedDateIndex: INITIAL_SELECTED_DATE_INDEX,
-        showBackButton: false,
-        jurisdictionToShow: {
-          category: 'fullCountry',
-          name: 'USA',
-          eventValue: null,
-        },
-        mapObject: null,
-        showSpinner: false,
-        viewMapTable: false,
-        exposureMapData: this.analyticsData.exposure[Number(INITIAL_SELECTED_DATE_INDEX)].value,
-        isolationMapData: this.analyticsData.isolation[Number(INITIAL_SELECTED_DATE_INDEX)].value,
-      };
+    this.state = { loading: true };
+  }
 
-      // A lot of times, the CountyLevelMaps functions will take time to render some new jurisdiction map.
-      // We want the Spinner to spin until they report back that they're done. Therefore, we set spinnerState to 2
-      // and subtract 1 every time they report back. When they each report back, 2 - 1 - 1 = 0 then we set `showSpinner` to false
-      // Because calls to setState are asynchronous, this value must be on the component itself
-      this.spinnerState = 0;
+  componentDidMount() {
+    axios.get('/analytics/monitoree_maps').then(response => {
+      const maxDaysOfHistory = response.data.monitoree_maps.length;
+      const selectedDateIndex = maxDaysOfHistory - 1;
+      const analyticsData = this.parseAnalyticsStatistics(response.data.monitoree_maps, maxDaysOfHistory);
+      if (analyticsData.exposure[Number(selectedDateIndex)]) {
+        this.setState({
+          analyticsData,
+          jurisdictionsPermittedToView: this.obtainJurisdictionsPermittedToView(response.data.monitoree_maps, maxDaysOfHistory),
+          jurisdictions_not_in_use: this.obtainJurisdictionsNotInUse(response.data.monitoree_maps, maxDaysOfHistory),
```

Comment:
Minor thing, but typically we use camelCase for the front-end (where possible)

Reply:
done: https://github.com/SaraAlert/SaraAlert/pull/836/commits/ff6f46ed0f97cc49a16082fc0abb3adb6edac11d